pagopa/io-app
# fix: [#177534705] Fixed header with close button that causes press problems

## Short description
Changed app header to fix layout and on press close button problems

## How to test
Move into the screen that shows the fiscal code in full screen, see if the close button is in correct position inside the header and if on press works properly

**Before changes:**
<img width="300" alt="Screenshot 2021-04-28 at 10 53 58" src="https://user-images.githubusercontent.com/61834253/116382539-4d9fd780-a816-11eb-9a20-cbee00cd69fe.png">

**After  changes:**
<img width="300" alt="Screenshot 2021-04-28 at 11 39 43" src="https://user-images.githubusercontent.com/61834253/116382711-7cb64900-a816-11eb-8f4d-90af1262f9a4.png">
